src/Security/RememberLoginHash.php

```diff
@@ -84,6 +84,33 @@ class RememberLoginHash extends DataObject
      */
     private $token = null;
 
+    /**
+     * Used to override the config value of logout_across_devices
+     * Tri-state where null denotes an unset override value
+     *
+     * @var bool|null
+     */
+    protected static $logoutAcrossDevices = null;
+
+    /**
+     * @return bool
+     */
+    public static function getLogoutAcrossDevices(): bool
+    {
+        if (is_bool(static::$logoutAcrossDevices)) {
+            return static::$logoutAcrossDevices;
+        }
+        return static::config()->get('logout_across_devices');
+    }
+
+    /**
+     * @param bool $value
```

Comment:
It's probably worth adding a comment here to explain what this does since an outside class calling it will probably not have the info attach to the protected variable.

Reply:
Updated

Reply:
Updated